inheritance_project/inheritance_project/urls.py

```diff
@@ -1,4 +1,4 @@
-"""inheritance_project URL Configuration
+inheritance_project URL Configuration
```

Comment:
It looks like you deleted the triple quote which begins a comment block and that means this will break because the comments on lines 1 through 14 will be interpreted as code.